tests/unittest/pywbem/test_subscriptionmanager.py

```diff
@@ -298,7 +300,7 @@ def test_create_owned_subscription(self):
 
             # confirm destination instance paths match
             assert sub_mgr.get_all_destinations(server_id)
-            # TODO: ks Finish this test completely when we add other
+            # ks Finish this test completely when we add other
```

Comment:
Nothing wrong with a TODO in test code. We only enforce no TODOs in pywbem code.